packages/quantic/cypress/integration/facets/timeframe-facet/timeframe-facet.cypress.ts

```diff
@@ -279,6 +279,30 @@ describe('quantic-timeframe-facet', () => {
       });
 
       scope('when entering an invalid range', () => {
+        scope('with empty start and end dates', () => {
+          Actions.submitForm();
+
+          Expect.numberOfValidationErrors(2);
+          Expect.validationError('Complete this field.');
+        });
+
+        scope('when entering then erasing dates', () => {
+          Actions.typeStartDate(validRange.start);
+          Actions.submitForm();
+
+          Expect.numberOfValidationErrors(1);
+          Expect.validationError('Complete this field.');
+
+          Actions.typeEndDate(validRange.end);
+          Actions.submitForm();
+          cy.wait(InterceptAliases.Search);
+
+          Expect.numberOfValidationErrors(0);
+
+          Actions.clearFilter();
+          cy.wait(InterceptAliases.Search);
```

Comment:
is there supposed to be a validation after this?

Reply:
No, it is simply there to reset the facet state.